src/Orleans.Runtime/Catalog/Catalog.cs

```diff
@@ -1443,5 +1444,15 @@ private void OnSiloStatusChange(SiloAddress updatedSilo, SiloStatus status)
                 }
             }
         }
+
+        private SiloAddress TryGetGrainDirectoryPartitionForDiagnostics(GrainId grainId)
```

Comment:
Nit: Isn't it more common for a `TryGetXX` to return a bool value and have an out parameter?

Reply:
Certainly is. I'll change it.